src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -3897,6 +3897,80 @@ def _edge_zone_model(cmd, edge_zone):
 
 
 # region LoadBalancers
+def create_load_balancer_latest(cmd, load_balancer_name, resource_group_name, location=None, tags=None,
```

Comment:
Similarly, with https://github.com/Azure/azure-cli/pull/24222, we don't need to create a new function for modification.